bigtable-hbase-integration-tests/src/test/java/com/google/cloud/bigtable/hbase/TestListTables.java

```diff
@@ -85,8 +88,34 @@ public void testTableNames() throws IOException {
         Assert.assertTrue(ArrayUtils.contains(tableList, tableName2));
       }
 
-      checkColumnFamilies(admin, tableName2);
+      {
+        TableName[] tableList = admin.listTableNames(Pattern.compile("list_table1-.*"));
+        Assert.assertTrue(ArrayUtils.contains(tableList, tableName1));
+        Assert.assertFalse(ArrayUtils.contains(tableList, tableName2));
```

Comment:
We have different names for the tables because we add a random UUID as a suffix.  We're retrieving based on a pattern with a shared prefix, but verify that the specific table is in the list.